drivers/linstorvolumemanager.py

```diff
@@ -785,6 +785,30 @@ def ensure_volume_list_is_not_locked(self, volume_uuids, timeout=None):
         if waiting:
             self._logger('No volume locked now!')
 
+    def make_volume_available(self, volume_uuid):
```

Comment:
Ok, I did miss, that a resource_create was "hidden" in the get_device_path.
I used make_available now instead of the resource_create (but they nearly do the same thing).